mne/viz/_3d.py

```diff
@@ -626,50 +624,35 @@ def plot_alignment(info, trans=None, subject=None, subjects_dir=None,
     show_axes : bool
         If True (default False), coordinate frame axis indicators will be
         shown:
-
         * head in pink
         * MRI in gray (if ``trans is not None``)
         * MEG in blue (if MEG sensors are present)
-
         .. versionadded:: 0.16
     fig : mayavi figure object | None
         Mayavi Scene (instance of mlab.Figure) in which to plot the alignment.
         If ``None``, creates a new 600x600 pixel figure with black background.
-
         .. versionadded:: 0.16
     interaction : str
         Can be "trackball" (default) or "terrain", i.e. a turntable-style
         camera.
-
         .. versionadded:: 0.16
     verbose : bool, str, int, or None
         If not None, override default verbose level (see :func:`mne.verbose`
         and :ref:`Logging documentation <tut_logging>` for more).
-
-    Returns
-    -------
-    fig : instance of mlab.Figure
-        The mayavi figure.
```

Comment:
if you don't return anything anymore it's an API break. Careful with this.

Reply:
what do you suggest to return then? `mlab.Figure` is backend specific.

Reply:
What do you think of returning the instance of the Renderer itself? That would be really convenient considering the current architecture.